Package.swift

```diff
@@ -1,33 +1,33 @@
-// Until Swift Package Manager has Resource Bundle support, Package.swift cannot be supported properly.
+// swift-tools-version:5.3
 
+//  Copyright © 2019 SpotHero, Inc. All rights reserved.
 
-// // swift-tools-version:5.0
+import PackageDescription
 
-// //  Copyright © 2019 SpotHero, Inc. All rights reserved.
-
-// import PackageDescription
-
-// let package = Package(
-//     name: "SpotHeroEmailValidator",
-//     platforms: [
-//         .iOS(.v8),          // minimum supported version via SPM
-//         .macOS(.v10_10),    // minimum supported version via SPM
-//         .tvOS(.v9),         // minimum supported version via SPM
-//         // watchOS is unsupported
-//     ],
-//     products: [
-//         .library(name: "SpotHeroEmailValidator", targets: ["SpotHeroEmailValidator"]),
-//     ],
-//     targets: [
-//         .target(
-//             name: "SpotHeroEmailValidator",
-//             dependencies: []
-//         ),
-//         .testTarget(
-//             name: "SpotHeroEmailValidatorTests",
-//             dependencies: [
-//                 .target(name: "SpotHeroEmailValidator"),
-//             ]
-//         ),
-//     ]
-// )
+let package = Package(
+    name: "SpotHeroEmailValidator",
+    platforms: [
+        .iOS(.v8),          // minimum supported version via SPM
```

Comment:
Can we bump this to v9 to avoid Xcode 12 warnings?
```suggestion
        .iOS(.v9),          // minimum supported version via SPM
```

Reply:
Yeah, was going to bump this with the other OS in the next few commits, but no harm in doing this now.